tests/light/src/syslog_ng/syslog_ng.py

```diff
@@ -25,7 +25,7 @@
 
 
 class SyslogNg(object):
-    def __init__(self, instance_paths, testcase_parameters):
+    def __init__(self, instance_paths, testcase_parameters, fd_close_callback):
```

Comment:
this variable is not stored anywhere.

Reply:
Thank you very much for the review comment. 
Yes, sorry, it was stored in an other commit. With editing commits I will fix this too.

Reply:
In my last push I have fixed this